x-pack/plugins/maps/public/classes/layers/layer.tsx

```diff
@@ -400,7 +400,11 @@ export class AbstractLayer implements ILayer {
   }
 
   isLayerLoading(): boolean {
-    return this._dataRequests.some((dataRequest) => dataRequest.isLoading());
+    const areTilesLoading =
+      typeof this._descriptor.__areTilesLoaded === 'boolean'
+        ? this._descriptor.__areTilesLoaded
```

Comment:
It might make more sense to negate this line instead of on line 407. This way it's clear that `areTilesLoading` is the boolean opposite state of `this._descriptor.__areTilesLoaded`